.eslintrc.json

```diff
@@ -8,6 +8,11 @@
     "node": true,
     "mocha": true
   },
+  "settings": {
```

Comment:
gets rid of a warning about not being able to detect the react version during linting. The react linter gets pulled in transitively even though we don't use it, and this is the magic incantation to silence that warning.